File: pkg/java/src/main/java/dev/openfga/language/validation/RewriteType.java
package dev.openfga.language.validation;

public enum RewriteType {
    Direct("direct"),
    ComputedUserset("computed_userset"),
    TupleToUserset("tuple_to_userset");

    private final String value;

    RewriteType(String value) {
        this.value = value;
    }
}

File: pkg/csharp/src/OpenFga.Language/Validation/RewriteType.cs
using System.Text.Json.Serialization;

namespace OpenFga.Language.Validation;

public enum RewriteType {
    [JsonPropertyName("direct")]
    Direct,

    [JsonPropertyName("computed_userset")]
    ComputedUserset,

    [JsonPropertyName("tuple_to_userset")]
    TupleToUserset
}